<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>欢迎登录智能旅游系统</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Button_1">
     <property name="text">
      <string>1.设计一条到若干个城市旅游并返回的路线</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Button_2">
     <property name="text">
      <string>2.设计到某个城市旅游的最优路线</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Button_3">
     <property name="text">
      <string>3.根据预算智能推荐路线</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Button_exit">
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
